frontend/app/data.tsx

```diff
@@ -41,6 +41,9 @@ export interface User {
   id: number;
   username: string;
   password: string;
+
+  //Backend part
+  backendCookie?: string;
```

Comment:
```suggestion
  backendCookie?: string[];
```
store cookies as array and not one string